python/src/FunctionalChaosSobolIndices.i

```diff
@@ -8,3 +8,109 @@
 
 %include openturns/FunctionalChaosSobolIndices.hxx
 namespace OT { %extend FunctionalChaosSobolIndices { FunctionalChaosSobolIndices(const FunctionalChaosSobolIndices & other) { return new OT::FunctionalChaosSobolIndices(other); } } }
+
+%pythoncode %{
+import math
+def __FunctionalChaosSobolIndices_repr_html(self):
+    """Get HTML representation."""
+    chaosResult = self.getFunctionalChaosResult()
+    inputDimension = chaosResult.getDistribution().getDimension()
+    outputDimension = chaosResult.getMetaModel().getOutputDimension()
+    enumerateFunction = chaosResult.getOrthogonalBasis().getEnumerateFunction()
+    indices = chaosResult.getIndices()
+    basisSize = indices.getSize()
+    coefficients = chaosResult.getCoefficients()
+    inputDescription = chaosResult.getDistribution().getDescription()
+    sobolIndexFormat = openturns.ResourceMap.GetAsString("FunctionalChaosSobolIndices-PrintFormat")
+    partOfVarianceThreshold = openturns.ResourceMap.GetAsScalar("FunctionalChaosSobolIndices-VariancePartThreshold")
+    maximumNumberOfOutput = openturns.ResourceMap.GetAsUnsignedInteger("FunctionalChaosSobolIndices-MaximumNumberOfOutput")
+    
+    # compute the mean, the variance, the standard deviation
+    mean = openturns.Point(outputDimension)
```

Comment:
Using `FunctionalChaosRandomVector`, this reduces to 2 lines, as already done in the C++ pretty-print.

Reply:
Fixed at 8e59da574.